docs/admin/compose-installation-guide.md

```diff
@@ -499,19 +503,22 @@ version: '3.7'
 secrets:
   lims_password:
     file: ${MISO_DB_PASSWORD_FILE}
+  root_password:
+    file: ${MISO_DB_PASSWORD_FILE}
   ssl_password:
     file: ${SSL_PASSWORD_FILE}
 
 services:
   db:
-    image: mysql:5.7.25
+    image: mysql:5.8
```

Comment:
`5.8` -> `8.0`?

Reply:
fixed

Reply:
Perhaps it got fixed elsewhere? Still looks to be `5.8` here in this file